src/breadcrumb-group/__tests__/breadcrumb-item.test.tsx

```diff
@@ -30,6 +32,16 @@ const items = [
   },
 ];
 describe('BreadcrumbGroup Item', () => {
+  let originalMatchMedia: typeof window['matchMedia'];
+
+  beforeAll(() => {
+    originalMatchMedia = window.matchMedia;
+  });
+
+  afterAll(() => {
+    window.matchMedia = originalMatchMedia;
+  });
```

Comment:
This is also a bit of a "meh" cleanup attempt for this approach since all the tests passed without this being here. 

Is it needed? - ¯\_(ツ)_/¯. 
Do I feel better knowing the original function is restored - yes.